common/core/desktop/tests/unit/rust_mock/state_api.cpp

```diff
@@ -165,7 +165,7 @@ int main(int, char * [])
   if (attrs->max_context < 16) return __LINE__;
 
   try_status(km_kbp_process_event(test_state, KM_KBP_VKEY_S,
-                                  KM_KBP_MODIFIER_SHIFT));
+                                  KM_KBP_MODIFIER_SHIFT, 1));
```

Comment:
Similarly, we need to make sure we pass in keydown and keyup for this test.

Reply:
done